src/main/java/org/mastodon/mamut/classification/util/Classification.java

```diff
@@ -201,6 +216,48 @@ public void updateClusterNames()
 		} );
 	}
 
+	/**
+	 * Exports the classification data to a CSV file.
+	 * <br>
+	 * This method creates a CSV file with the classification data, including cell names, tag labels, class names, and class similarity scores.
+	 * The CSV file is created with a header and ";" as delimiter.
+	 *
+	 * @param file The file to which the CSV data will be written.
+	 * @param tagSet The tag set used for classification, which may be null.
+	 */
+	public void exportCsv( final File file, final TagSetStructure.TagSet tagSet )
+	{
+		char separator = ';';
+		String[] header = new String[] { "Cell name", tagSet == null ? "" : tagSet.getName(), "Class", "Class similarity score" };
+		CSVFormat csvFormat = CSVFormat.Builder.create().setHeader( header ).setDelimiter( separator ).setQuote( '"' ).setEscape( '"' )
+				.setRecordSeparator( "\n" ).build();
+		try (
+				FileWriter writer = new FileWriter( file );
+				CSVPrinter csvPrinter = new CSVPrinter( writer, csvFormat ))
+		{
+			for ( Classification.ObjectClassification< T > objectClassification : getObjectClassifications() )
+			{
+				for ( T object : objectClassification.getObjects() )
+				{
+					String name = object instanceof HasName ? ( ( HasName ) object ).getName() : object.toString();
+					String tagLabel = object instanceof BranchSpotTree ? ( ( BranchSpotTree ) object ).getTagLabel() : "";
+					String className = objectClassification.getName();
+					String similarity = String.valueOf(
+							MathUtils.roundToSignificantDigits( objectClassification.getCluster().getDistance().getDistance(), 2 ) );
```

Comment:
Why is the number of significant digits 2?

I don't understand this deeply but lets imagine a tree with 1000 spots. And there is on spot removed, the similarity will be 1.0 ?? Meaning they are identical.

Reply:
I noticed that the similarity would be close to zero. So it would be shown with a higher precision. Which is good.

Reply:
After a discussion, I decide to let the method MathUtils.roundToSignificantDigits() return a String that is already formatted as specified by the parameters of the method.